### PR TITLE
add `lilypond_notation()` method to `Chord` class

### PR DESCRIPTION
**Pull request by mystery-house**
This PR adds a new `lilypond_notation()` method to the `Chord` class, as well as related unit tests.

It is analagous to the existing `Note.lilypond_notation()` method, but it formats the chord for use in LilyPond's [\chordmode](https://lilypond.org/doc/v2.23/Documentation/notation/chord-mode) context, taking into account differences between chord modifiers used by Musthe vs those used by LilyPond.

**Examples:**
```python
>>> Chord(Note('C')).lilypond_notation()
'c'
>>> Chord(Note('G#'), 'dom7').lilypond_notation(4)
'gis4:7'
>>> Chord(Note('Eb'), 'open5').lilypond_notation('8.')
'ees8.:1.5.8' 
```

### CODE REVIEW

File: musthe/musthe.py [hunk 1]
```diff
@@ -332,6 +333,20 @@ class Chord:
...
+        'min': 'm',
+        'dom7': '7',
+        'min7': 'm7',
+        'm7dim5': '7.5-',
+        'open5':  '1.5.8',        
+    }
+    """Maps Musthe chord modifiers to their LilyPond `\chordmode` equivalents. Any items
+    omitted from this list (``min``, ``aug`` and ``dim``, for example) are
+    assumed to use the same value in LilyPond.
+    """
```
Comment (gciruelos) [inline musthe/musthe.py:348]: can you move this comment above the variable declaration?

File: musthe/musthe.py [hunk 2]
```diff
@@ -379,6 +394,30 @@ def __eq__(self, other):
         else:
             return all(self.notes[i] == other.notes[i]
                        for i in range(len(self.notes)))
+    
+    def lilypond_notation(self, duration: Union[str,int] = ""):
```
Comment (gciruelos) [inline musthe/musthe.py:398]: nit: add a comma: Union[str, int]

File: musthe/musthe.py [hunk 3]
```diff
@@ -379,6 +394,30 @@ def __eq__(self, other):
...
+        """Returns the chord as string that can be used in a LilyPond
+        ``\chordmode`` block.
```
Comment (gciruelos) [inline musthe/musthe.py:400]: can you document what the "duration" parameter means?

File: tests/tests.py [hunk 4]
```diff
@@ -287,6 +287,43 @@ def test_chord_repr(self):
...
+    def test_lilypond_mod_diff(self):
+        """Tests LilyPond chord notation where the Musthe and LilyPond chord
+        modifiers are different.
+        """
+        self.assertEqual(Chord(Note('C'), 'min').lilypond_notation(), 'c:m')
+        self.assertEqual(Chord(Note('C'), 'dom7').lilypond_notation(), 'c:7')
+        self.assertEqual(Chord(Note('C'), 'min7').lilypond_notation(), 'c:m7')
+        self.assertEqual(Chord(Note('C'), 'm7dim5').lilypond_notation(4), 'c4:7.5-')
+        self.assertEqual(Chord(Note('C'), 'open5').lilypond_notation("16."), 'c16.:1.5.8')
+
```
Comment (gciruelos) [inline tests/tests.py:326]: can you add a test that the keys of Chords.recipes and the keys of Chords.lilypond_modifiers match, except for the known missing ones?

**Review by gciruelos — CHANGES_REQUESTED** [review]
Overall LGTM, just left some comments.

File: musthe/musthe.py (hunk 1 above)
Comment (mystery-house) [reply musthe/musthe.py:348]: Will do. (I was surprised to find that the "official" best practice for attribute docstrings is putting them after the attribute, but I'm not crazy about it either)

Comment (mystery-house) [reply musthe/musthe.py]: This should be fixed in the updated pr.

File: musthe/musthe.py (hunk 2 above)
Comment (mystery-house) [reply musthe/musthe.py]: This should be fixed in the updated pr.

File: musthe/musthe.py (hunk 3 above)
Comment (mystery-house) [reply musthe/musthe.py:400]: This should be fixed in the updated pr.

File: tests/tests.py (hunk 4 above)
Comment (mystery-house) [reply tests/tests.py:339]: `test_lilypond_mod_same` has been extended to cover all of the chord recipes where the musthe name is the same as the lilypond name.

I also extended `test_lilypond_accidentals` to test all possible accidental notes, and I added `test_lilypond_aliases` to test all possible musthe chord aliases.

**Pull request merged**